packages/features/users/components/UserTable/ChangeUserRoleModal.tsx

```diff
@@ -0,0 +1,27 @@
+import { useSession } from "next-auth/react";
+import type { Dispatch } from "react";
+
+import MemberChangeRoleModal from "@calcom/features/ee/teams/components/MemberChangeRoleModal";
+
+import type { Action, State } from "./UserListTable";
+
+export function ChangeUserRoleModal(props: { state: State; dispatch: Dispatch<Action> }) {
```

Comment:
Wrapper around MemberChangeRoleModel that allows us to pass in as little props as possible